html_css/web_accessibility.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+
+At this point in the curriculum you've learned incredibly valuable concepts in whichever full stack path you chose, you've created some awesome, resume worthy projects, and you should have a better understanding of some HTML and CSS concepts. One could even argue that you may now love working with CSS...?
+
+What you may not have too much an understanding of, though, is the topic of accessibility, often referred to as "a11y" (due to there being 11 letters between the first and last letters). Unfortunately, this is a topic that many people either don't know much about, or just don't take into account when developing websites. If you fit into either of those two categories, you may have adopted some habits that aren't exactly a11y friendly. Before we dive into how you can break away from such habits and make more a11y friendly websites, it's important to first understand some basic information when it comes to web accessibility.
+
+### Learning Outcomes
+By the end of this lesson, you should be able to:
+
+* Explain what web accessibility is.
+* Explain the purpose of the Web Content Accessibility Guidelines.
+* Name the 4 principles of the WCAG.
+* Know the differences between the WCAG conformity levels.
+
+### What is Web Accessibility?
+
+Web accessibility is when websites don't have barriers that prevent interaction, access, or contribution from users with a disability, situational limitation, or changing abilities due to age.
+
+There are different types of disabilities, including (but not limited to) auditory, physical/motor, cognitive, or visual. A disability can be permanent, such as a user who is completely blind, or it can be temporary, such as a user with a broken arm, and users may have multiple disabilities. Older users with changing abilities may even have similar conditions as those who have a disability.
+
+Situational limitations are a little different. Using a phone outside on a bright day, browsing a site with one hand while you're busy doing something else with the other, or living in an area where the internet is slow or the bandwidth is expensive are all examples of situational limitations. These limitations, unlike a disability, occur to users only in specific situations, but are still important to keep in mind when developing a website.
+
+### Why Web Accessibility Matters
+
```

Comment:
I wonder if we should provide a more general, non-web related example of "accessibility" to introduce the point below.

Reply:
Could you clarify on this one a little more? Wasn't sure if you meant the "Why Web Accessibility Matters" or the "situational limitations" part for providing a non-web related example.

Reply:
This comment was intended for the section of "Why Web Accessibility Matters"

Reply:
Ah gotcha. So perhaps instead of listing only web-related examples in the opening paragraph of that section ("text was too difficult to read..."), it might be good to open that section up with non-web examples that could maybe relate more broadly to people?

I will comment that my intention was to keep things web related as much as possible, but if the above is along the lines of what you mean then I could definitely see grounds for making that sort of change.

Reply:
Correct.

The reason being with the given example, the difference between "annoying" and "impossible" isn't yet clear for why the problems mentioned would make it impossible.

Reply:
This is just off the top of my head currently:

"Imagine being in a multi-story building that has no elevator. For some people, this might only be an annoyance. "Huh, no elevator. I guess I'll walk up two flights of stairs, then." A person that requires a wheelchair, however, would find it impossible, or at the very least much more difficult, to go anywhere beyond the first floor. Even if a person in a wheelchair had someone to lift the wheelchair up each step, it would be a much more difficult process. The point is that an elevator would have made this building more accessible."